testspec/src/09_pbkdf.rst

```diff
@@ -95,3 +95,63 @@ test vectors are listed in :srcref:`src/tests/data/pbkdf/pbkdf2.vec`.
    |                        |    the PBKDF2 and compare the result with the expected output value     |
    |                        |    *Out*                                                                |
    +------------------------+-------------------------------------------------------------------------+
+
+
+Argon2
+------
+
+Argon2 with its configurations Argon2, Argon2i, Argon2d and Argon2id is tested with the following constraints:
```

Comment:
What do you mean with *Argon2 configuration*? Should there not be only the three `i`, `d` and `id` variants?

Reply:
True. Good catch.